integration-tests/timeSheetEntryCollection/odata-client/manage-workforce-timesheet-service/BatchRequest.d.ts

```diff
@@ -0,0 +1,21 @@
+/*!
```

Comment:
As discussed, the service cannot be exposed here. Please remove it and add instructions for the user on how to do this instead.